packages/generator-widget/CHANGELOG.md

```diff
@@ -6,6 +6,15 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ## [Unreleased]
 
+## [9.20.0] - 2022-12-01
```

Comment:
I just noticed this: the version headers in the changelog are added as part of the release process, when everything under `[Unreleased]` gets moved to a header for that specific release. Adding this header manually (with a somewhat arbitrary prediction for a release date ;)) breaks that process, please remove it :).

@jeffersonRibeiro This reminds me that I'm not seeing any release notes for `assignableTo` under `Unreleased`, we definitely need to fix that before the 9.20 release of the tools as well 🙈.

Reply:
@tamelbrun fixed